<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QGroupBox" name="grpbx">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QToolButton" name="tlbttnLanzar">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tlbttnExportar">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tlbttnSalir">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tlbttnAyuda">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>38</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="grpBxConsole">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTextEdit" name="txtEdtConsole"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionSalir">
   <property name="icon">
    <iconset resource="resources/actionIcos.qrc">
     <normaloff>:/icos32/icos32/system-log-out.png</normaloff>:/icos32/icos32/system-log-out.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Salir</string>
   </property>
   <property name="toolTip">
    <string>Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionExportar">
   <property name="icon">
    <iconset resource="resources/actionIcos.qrc">
     <normaloff>:/icos32/icos32/document-save-as.png</normaloff>:/icos32/icos32/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exportar</string>
   </property>
   <property name="toolTip">
    <string>Exportar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionAyuda">
   <property name="icon">
    <iconset resource="resources/actionIcos.qrc">
     <normaloff>:/icos32/icos32/help-browser.png</normaloff>:/icos32/icos32/help-browser.png</iconset>
   </property>
   <property name="text">
    <string>Ay&amp;uda</string>
   </property>
   <property name="toolTip">
    <string>Ayuda</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionLanzarApp">
   <property name="icon">
    <iconset resource="resources/actionIcos.qrc">
     <normaloff>:/icos32/icos32/applications-internet.png</normaloff>:/icos32/icos32/applications-internet.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Lanzar App</string>
   </property>
   <property name="toolTip">
    <string>Lanzar App</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources/actionIcos.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>on_lanzarApp_triggered()</slot>
 </slots>
</ui>
